src/client/snippets/paportal-liquid.code-snippets

```diff
@@ -88,6 +133,23 @@
 		],
 		"description": "Fully renders a web form by its name"
 	},
+    "Code-Component": {
+        "prefix": "code-component",
+        "body": [
+			"{% codecomponent name:\"$1\" controlValue:\"$2\" controlApiKey:\"${3:<API Key Value here>}\" %}"
+        ],
+        "description": "Adds an Iframe component"
```

Comment:
Please ask Neeraj to review these strings.

Reply:
Synced with Neeraj, will raise another PR for string changes